src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -85,78 +37,124 @@ const ArtistAuctionResultItemTop: FC<Props> = props => {
-          )}
-        </ResponsiveBox>
-      </Column>
+    <Box width="100%">
+      <GridColumns>
+        <Column span={[4, 2]}>
+          <ResponsiveBox
+            aspectWidth={1}
+            aspectHeight={1}
+            maxWidth={130}
+            bg="black10"
+          >
+            {image ? (
+              <Image
+                src={image.src}
+                srcSet={image.srcSet}
+                width="100%"
+                height="100%"
+                alt=""
+                lazyLoad
+              />
+            ) : (
+              <Box
+                display="flex"
+                justifyContent="center"
+                alignItems="center"
+                height="100%"
+              >
+                <NoArtworkIcon height={24} width={24} fill="black60" />
+              </Box>
+            )}
+          </ResponsiveBox>
+        </Column>
+
+        <Column span={[8, 1]} display={["block", "none"]}>
+          <Box>
+            {!!showArtistName && <Text variant="xs">{artistName}</Text>}
+
+            <Text variant="xs" lineClamp={1}>
+              {[title, date_text].filter(Boolean).join(", ")}
+            </Text>
+
+            {mediumText !== "Unknown" && (
+              <Text variant="xs" color="black60" lineClamp={1}>
+                {mediumText}
+              </Text>
+            )}
+
+            {dimension_text && (
+              <Text variant="xs" color="black60" lineClamp={1}>
+                {dimension_text}
+              </Text>
+            )}
+
+            <Spacer y={1} />
+
+            <Text variant="xs" color="black60" lineClamp={1}>
+              {dateOfSale} • {organization}
+            </Text>
+
+            <ArtistAuctionResultItemPrice {...props} />
+          </Box>
+        </Column>
 
-      <Column span={4}>
-        {!!showArtistName && <Text variant="sm-display">{artistName}</Text>}
+        <Column span={4} display={["none", "block"]}>
+          {!!showArtistName && <Text variant="sm-display">{artistName}</Text>}
 
-        {showArtistName ? (
-          <Text variant="sm-display" color="black60">
+          <Text variant="sm-display">
             <i>{title?.trim()}</i>
             {date_text &&
               date_text.replace(/\s+/g, "").length > 0 &&
               ", " + date_text}
           </Text>
-        ) : (
-          <Text variant="sm-display">
-            {[title, date_text].filter(Boolean).join(", ")}
-          </Text>
-        )}
 
-        {mediumText !== "Unknown" && (
-          <Text variant="xs" color="black60" lineClamp={4}>
-            {mediumText}
-          </Text>
-        )}
-      </Column>
+          {mediumText !== "Unknown" && (
+            <Text variant="xs" color="black60">
+              {mediumText}
+            </Text>
+          )}
 
-      <Column span={2}>
-        <Text variant="sm-display">{dateOfSale}</Text>
+          {dimension_text && (
+            <Text variant="xs" color="black60">
+              {dimension_text}
```

Comment:
**nitpick:** We should try to eliminate this renaming within the fragments. I think they are from another time 🧓